composer.json

```diff
@@ -14,7 +14,7 @@
         "acquia/acsf-contenthub-console": "dev-main",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "spatie/ssl-certificate": "1.21|1.22.1",
+        "spatie/ssl-certificate": "1.21|1.22.1|^2",
```

Comment:
1.21 is needed to support PHP 7.3 as D9.2 supports PHP7.3.
1.22.1 added support for both PHP8 along with PHP7.4 compatibility.
2+ releases dropped support for PHP7 but have added features.

Reply:
I think that could have been just ^1.21 I feel like, but that's all right for now.